stdlib/public/Darwin/Foundation/Foundation.swift

```diff
@@ -29,7 +29,153 @@ public let NSNotFound: Int = .max
+/// the `importLocalizations` option.
+///
+///     xcodebuild -importLocalizations -project <projectname>.xcodeproj \
+///                                     -localizationPath <path>
+///
+/// Choose Meaningful Keys
+/// ----------------------
+///
+/// Words can often have multiple different meanings depending on the context
+/// in which they're used. For example, the word "Book" can be used as a noun—a
+/// printed literary work—and it can be used as a verb—the action of making a
+/// reservation. Words with different meanings which share the same spelling are
+/// heteronyms.
+///
+/// Different languages often have different heteronyms. "Book" in English is
+/// one such heteronym, but that's not so in French, where the noun translates
+/// to "Livre", and the verb translates to "Réserver". For this reason, it's
+/// important make sure that each use of the same phrase is translated
+/// appropriately for its context by assigning unique keys to each phrase and
+/// adding a description comment describing how that phrase is used.
+///
+///     NSLocalizedString("book-tag-title", value: "Book", comment: """
+///     noun: A label attached to literary items in the library.
+///     """)
+///
+///     NSLocalizedString("book-button-title", value: "Book", comment: """
+///     verb: Title of the button that makes a reservation.
+///     """)
+///
+/// Use Only String Literals
+/// ------------------------
+///
+/// String literal values must be used with `key`, `tableName`, `value`, and
+/// `comment`.
+///
+/// Xcode does not evaluate interpolated strings and string variables when
+/// generating strings files from code. Using those language features will
+/// result in exported localizations that resemble the code expression instead
+/// of its value. Translators would then translate that exported value—leaving
+/// international users with a localized string containing code.
+///
+///     // Translators will see "1 + 1 = (1 + 1)".
+///     // International users will see a localization "1 + 1 = (1 + 1)".
+///     let localizedString = NSLocalizedString("string-interpolation",
+///                                             value: "1 + 1 = \(1 + 1)"
+///                                             comment: "A math equation.")
+///
+/// To dynamically insert values within localized strings, set `value` to a
+/// format string, and use `String.localizedStringWithFormat(_:_:)` to insert
+/// those values.
+///
+///     // Translators will see "1 + 1 = %d" (they know what "%d" means).
+///     // International users will see a localization of "1 + 1 = 2".
+///     let format = NSLocalizedString("string-literal",
+///                                    value: "1 + 1 = %d",
+///                                    comment: "A math equation.")
+///     let localizedString = String.localizedStringWithFormat(format, (1 + 1))
+///
+/// Multi-line string literals are technically supported, but will result in
```

Comment:
```suggestion
/// Multiline string literals are technically supported, but will result in
```
Here and below, too…